modules/claims_api/app/models/claims_api/power_of_attorney.rb

```diff
@@ -53,6 +118,18 @@ def external_uid
       source_data.present? ? source_data['icn'] : Settings.bgs.external_uid
     end
 
+    def signature_image_paths
+      @signature_image_paths ||= {}
+    end
+
+    def create_signature_image(signature_type)
+      path = "/tmp/#{signature_type}_#{id}_signature.png"
```

Comment:
same concern https://github.com/department-of-veterans-affairs/vets-api/pull/4428/files#r444194792